network-api/networkapi/urls.py

```diff
@@ -30,6 +30,9 @@
 
 urlpatterns = list(filter(None, [
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^soc/', include('social_django.urls', namespace='social'))
+    if settings.SOCIAL_SIGNIN else '',
+    url(r'^fellowships/', include('networkapi.fellows.urls')),
```

Comment:
on line 47 below, there's this code:

```python
url(r'^soc/', include('social_django.urls', namespace='social')),
 
# Don't remove this redirect until Fellowships pages are live
url(r'^fellowships/$', RedirectView.as_view(
             url='https://advocacy.mozilla.org/open-web-fellows'
         ))
if settings.SOCIAL_SIGNIN else None,
```

There are two things we need to look at here:
Firstly, we'll want to update the route pattern to something like this
```python
url(r'^fellowship/(?P<path>.*)', RedirectView.as_view(
    url='/fellowships/%(path)s',
    query_string=True
)),
```
Secondly the line with 
```python
if settings.SOCIAL_SIGNIN else None,
``` 
should actually follow 
```python
url(r'^soc/', include('social_django.urls', namespace='social'))
```
... but there are two of the above url configs in the file, so lets de-dupe that!